src/Tribe/Main.php

```diff
@@ -3891,225 +3891,6 @@ public function isOrganizer( $postId = null ) {
 			return false;
 		}
 
-		/**
-		 * Modify the WHERE clause of query when fetching next/prev posts so events with identical times are not excluded
-		 *
-		 * This method ensures that when viewing single events that occur at a given time, other events
-		 * that occur at the exact same time are are not excluded from the prev/next links
-		 *
-		 * @since 4.0.2
-		 *
-		 * @param string $where_sql WHERE SQL statement
-		 * @param WP_Query $query WP_Query object
-		 *
-		 * return string
-		 */
-		public function get_closest_event_where( $where_sql ) {
```

Comment:
Don't remove the method, move them to the  Deprecated section and throw a `_deprecated_function()`